ivy/functional/backends/torch/experimental/layers.py

```diff
@@ -927,7 +927,11 @@ def adaptive_avg_pool1d(input, output_size):
 
 
 @with_unsupported_dtypes({"2.0.1 and below": ("bfloat16", "float16")}, backend_version)
-def adaptive_avg_pool2d(input, output_size):
+def adaptive_avg_pool2d(
+    input,
+    output_size,
+    data_format: str = "NCHW",
+):
     return torch.nn.functional.adaptive_avg_pool2d(input, output_size)
```

Comment:
You don't handle `data_format` at all here. And you also haven't added it to `adaptive_avg_pool1d`. Please complete the code and make sure to run the tests locally to confirm the changes work.

@vedpatwardhan there must be a problem in the CI checks since the test is shown to be passing.

Reply:
Yeah there have been a few issues with the intelligent tests, but the `adaptive_avg_pool2d` test seems to have [run](https://github.com/unifyai/ivy/actions/runs/6160405092/job/16717945740?pr=23065#step:3:8017), so the problem isn't with the discovery of tests which is the main problem in most other PRs. I'm not sure why the test is passing though, does it fail when you run the test with `25` examples (as that's the max_examples used in the intelligent tests)? Thanks @AnnaTz 😄

Reply:
Yes, it fails locally.